File: extract-code-examples/tests/fixtures/typescript/src/api/resources/fhir/types/FhirPatchRequestBodyItem.ts
// Synthetic fixture: minimal JSON Patch operation type. Real Fern output
// is richer; the schema-extraction tests don't require the variant set,
// only that the interface exists so nested-type resolution finds it.

export interface FhirPatchRequestBodyItem {
    op: string;
    path: string;
    value?: unknown;
}

File: extract-code-examples/tests/fixtures/java-schema/src/main/java/com/phenoml/api/resources/fhir/types/FhirPatchRequestBodyItem.java
/**
 * Synthetic fixture: minimal JSON Patch operation type. The schema
 * extractor only needs the class to exist + parse — none of its fields
 * are read during the passthrough-body tests.
 */
package com.phenoml.api.resources.fhir.types;

public final class FhirPatchRequestBodyItem {
    private final String op;
    private final String path;
}
